mds/models.py

```diff
@@ -69,7 +70,17 @@ def short_uuid4(uid):
     return str(uid)[:8]  # Basically splitting on the first dash
 
 
+def colors_validator(colors):
+    fields = ["primary", "secondary"]
+    for key in colors:
+        if key in fields:
```

Comment:
You mean `not in in fields` no ? I guess a test could be nice ☺︎ . You could also check value format (6/8 hex ?)

Reply:
ou
```python
if not set(colors.keys()).is_subset({"primary", "secondary"}):
```

Reply:
Nitpicking:
```python
fields = {"primary", "secondary"}
if not set(keys).issubset(fields):
    raise ...
```

Reply:
> You mean `not in in fields` no ? I guess a test could be nice ☺︎ . You could also check value format (6/8 hex ?)

I don't think the check on the hex format is necessary. after all you just set the colors once and for all

Reply:
> You mean `not in in fields` no ? I guess a test could be nice ☺︎ . You could also check value format (6/8 hex ?)

Hmm I only found checks using custom regular expressions, so no real tested function. Not sure if it is good to check that way.